lib/codeowners/cli/main.rb

```diff
@@ -13,19 +13,22 @@ module Codeowners
   module Cli
     # Command Line Interface used by bin/codeowners-checker.
     class Main < Base # rubocop:disable Metrics/ClassLength
+      include InteractiveHelpers
+
       option :from, default: 'origin/master'
       option :to, default: 'HEAD'
       option :interactive, default: true, type: :boolean, aliases: '-i'
       option :validateowners, default: true, type: :boolean, aliases: '-v'
       option :autocommit, default: false, type: :boolean, aliases: '-c'
       desc 'check REPO', 'Checks .github/CODEOWNERS consistency'
       # for pre-commit: --from HEAD --to index
-      def check(repo = '.')
+      def check(repo = '.') # rubocop:disable Metrics/MethodLength
         @content_changed = false
         @repo = repo
         setup_checker
         @owners_list_handler = OwnersListHandler.new
         @owners_list_handler.checker = @checker
+        @owners_list_handler.options = options
```

Comment:
You can pass `options` to the initializer, since OwnersListHandler is inherited from `Codeowners::Cli::Base`
```
@owners_list_handler = OwnersListHandler.new(args, options)
@owners_list_handler.checker = @checker
```

Reply:
thanks!

Reply:
Please also consider re-running rubocop - it seems, those comments are unwanted now.